prefect_databricks/flows.py

```diff
@@ -312,6 +305,191 @@ def jobs_runs_submit_and_wait_for_completion_flow(notebook_path, **base_paramete
+            jar_params. Use Task parameter variables to set parameters containing
+            information about job runs. The JSON representation of this field
+            (for example {"notebook_params":{"name":"john doe","age":"35"}}) cannot
+            exceed 10,000 bytes.
+        python_params (Optional[List[str]], optional):
+            A list of parameters for jobs with Python tasks, for example "python_params"
+            :["john doe", "35"]. The parameters are passed to Python file as command-
+            line parameters. If specified upon run-now, it would overwrite the
+            parameters specified in job setting. The JSON representation of this field
+            (for example {"python_params":["john doe","35"]}) cannot exceed 10,000 bytes
+            Use Task parameter variables to set parameters containing information
+            about job runs. These parameters accept only Latin characters (ASCII
+            character set). Using non-ASCII characters returns an error. Examples of
+            invalid, non-ASCII characters are Chinese, Japanese kanjis, and emojis.
+        spark_submit_params (Optional[List[str]], optional):
+            A list of parameters for jobs with spark submit task, for example
+            "spark_submit_params": ["--class", "org.apache.spark.examples.SparkPi"].
+            The parameters are passed to spark-submit script as command-line parameters.
+            If specified upon run-now, it would overwrite the parameters specified in
+            job setting. The JSON representation of this field (for example
+            {"python_params":["john doe","35"]}) cannot exceed 10,000 bytes.
+            Use Task parameter variables to set parameters containing information about
+            job runs. These parameters accept only Latin characters (ASCII character
+            set). Using non-ASCII characters returns an error. Examples of invalid,
+            non-ASCII characters are Chinese, Japanese kanjis, and emojis.
+        python_named_params (Optional[Dict], optional):
+            A map from keys to values for jobs with Python wheel task, for example
+            "python_named_params": {"name": "task", "data": "dbfs:/path/to/data.json"}.
+        pipeline_params (Optional[str], optional): Defaults to None.
+        sql_params (Optional[Dict], optional):
+            A map from keys to values for SQL tasks, for example "sql_params":
+            {"name": "john doe", "age": "35"}. The SQL alert task does not support
+            custom parameters.
+        dbt_commands (Optional[List], optional):
+            An array of commands to execute for jobs with the dbt task,
+            for example "dbt_commands": ["dbt deps", "dbt seed", "dbt run"]
+
+    Raises:
+        DatabricksJobTerminated: _description_
+        DatabricksJobSkipped: _description_
+        DatabricksJobInternalError: _description_
+
+    Returns:
+        Dict: _description_
```

Comment:
Ideally, we'd be more specific about the shape and content of the return value.
```suggestion
        Dict: A dictionary containing information about the completed job run.
```